monitoring/src/main/resources/web/js/init.js

```diff
@@ -5,6 +5,19 @@
 
     var graph = Viva.Graph.graph();
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ["Viva" is not defined.](https://www.codacy.com/app/lustefaniak/akka-viz/file/2100595375/issues/source?bid=3121282&fileBranchId=3142893#l6)